bentoml/deployment/serverless/aws_lambda.py

```diff
@@ -131,6 +132,11 @@ def generate_handler_py(bento_name, apis, output_path):
 
 class AwsLambdaDeploymentOperator(DeploymentOperatorBase):
     def apply(self, deployment_pb, repo, prev_deployment=None):
+        if which('docker') is None:
+            raise BentoMLMissingDepdencyException(
```

Comment:
this only test if docker client is installed, it may fail to connect to docker daemon, or docker daemon has not been started yet, let's add a common method for checking if docker is available?

Reply:
yeah, lets do that

Reply:
one potential way to do that is run 'docker info' and parse the output

Reply:
yeah, I was reading their documentation, that's the recommend way to test.  Also, I tested locally.  Using subprocess.check_output with `docker info`  if Docker is not running, it will throw exception